docs/domains/Domain.json

```diff
@@ -109,6 +109,10 @@
           "description": "The name of this cluster. Required",
           "type": "string"
         },
+        "allowReplicasBelowMinDynClusterSize": {
```

Comment:
Should this also be a global setting?  On the one hand, it's extra work and I'm not sure if the extra work is worth it at this time. On the other hand, there's precedent where 'replicas' itself is both global and per-cluster.  

One way to handle this would be check-in 'as-is' and file a JIRA to add a global setting...

Reply:
Filed [OWLS-81769](https://jira.oraclecorp.com/jira/browse/OWLS-81769)